src/vivarium_csu_alzheimers/data/loader.py

```diff
@@ -240,3 +245,128 @@ def reshape_to_vivarium_format(df, location):
     df = vi_utils.sort_hierarchical_data(df)
     df.index = df.index.droplevel("location")
     return df
+
+
+def load_alzheimers_all_states_prevalence(
+    key: str, location: str, years: int | str | list[int] | None = None
+) -> pd.DataFrame:
+    all_state_dur = load_alzheimers_duration_all_states(None, location, years)
+    alz_dur = load_alzheimers_duration(None, location, years)
+    alz_prev = get_data(data_keys.ALZHEIMERS.PREVALENCE, location, years)
+    return ((all_state_dur * alz_prev) / alz_dur).bfill(limit=2).fillna(0)
```

Comment:
add a comment explaining what `bfill(limit=2)` is intended to accomplish, in case we forget